apps/site/styles/effects.css

```diff
@@ -10,3 +10,47 @@ h1.special {
     md:leading-[4rem]
     md:-tracking-[0.06rem];
 }
+
+.turtle {
+  @apply animate-surf
+    absolute
+    z-10
+    translate-x-0
+    translate-y-0;
+}
+
+.turtle img {
+  @apply h-auto
+    w-[200px];
+}
+
+.turtle::after {
+  @apply absolute
+    left-[-50%]
+    top-[50%]
+    -z-10
+    block
+    h-[150px]
+    w-[150px]
+    rotate-[-90deg]
+    select-none
+    bg-[url('/static/images/smoke.gif')]
+    opacity-[0.15];
+  /* stylelint-disable-next-line declaration-empty-line-before */
+  content: '';
+}
+
+@media (max-width: 750px) {
+  .turtle {
+    @apply m-0;
+  }
+
+  .turtle img {
+    @apply w-[100px];
+  }
+
+  .turtle::after {
+    @apply left-[-100%]
+      top-[20%];
+  }
+}
```

Comment:
Since Tailwind responsive styling [utilities](https://tailwindcss.com/docs/responsive-design) implemented above, media query are no longer needed

```suggestion
```